internal/framework/service/infrastructure_access_target_deprecated/schema.go

```diff
@@ -0,0 +1,194 @@
+package infrastructure_access_target_deprecated
+
+import (
+	"context"
+
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/cloudflare/terraform-provider-cloudflare/internal/consts"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	dschema "github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+)
+
+func (r *InfrastructureAccessTargetDeprecatedResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		DeprecationMessage: "`cloudflare_infrastructure_access_target` is now deprecated and will be removed in the next major version. Use `cloudflare_zero_trust_infrastructure_access_target` instead.",
+		MarkdownDescription: heredoc.Doc(`
+			The [Infrastructure Access Target](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/use-cases/ssh/ssh-infrastructure-access/#4-add-a-target) resource allows you to configure Cloudflare Risk Behaviors for an account.
```

Comment:
this seems wrong? are we mixing up infrastructure targets and risk behaviours in this description?

Reply:
Thanks for catching this, will fix this soon!

Reply:
Fixed here @jacobbednarz -> [872a10a](https://github.com/cloudflare/terraform-provider-cloudflare/pull/4403/commits/872a10af63087d0d4880a3fdefd94f91a0421ed5)